charts/fluent-bit/templates/configmap-dashboards.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
   {{- end }}
   labels:
     {{- include "fluent-bit.labels" $ | nindent 4 }}
-    {{ $.Values.dashboards.labelKey }}: "1"
+    {{ $.Values.dashboards.labelKey }}: "{{ $.Values.dashboards.labelValue }}"
```

Comment:
```suggestion
    {{ .Values.dashboards.labelKey }}: {{ $.Values.dashboards.labelValue | quote }}
```
Sorry I missed this one earlier. The `quote` function is prefered over hardcoding and as we're not in a nested context we don't need to use `$`.